src/hid_freebsd.c

```diff
@@ -61,10 +48,6 @@ is_fido(int fd)
 	ugd.ugd_data = buf;
 	ugd.ugd_maxlen = sizeof(buf);
 
-	/*
```

Comment:
Wouldn't it make sense to leave the comment in (right now, fido_open is called first and does that, but maybe a future implementer does more optimizations for hidraw and would benefit from understanding about this side-effect?).

Reply:
We don't write to the fd in this code path (is_fido(), coming from fido_dev_info_manifest()). AFAICT, the fact we put hidraw(4) in uhid(4) compat mode is moot in this case, which is why I dropped the comment. Note that the compat mode is reset when we open the descriptor.

Reply:
Hm, we also use this code path coming from copy_info_*. You're right that currently it's not in the code path, but as the function cannot decide where it is called from, adding the comment might help in the future. Maybe that's also a philosophical question (like many things around code comments).

Reply:
I'll leave the comment out for now, as you seem confident and know the code base better than I do.